### ISSUE TITLE
dhcp hard coded

### ISSUE THREAD

**Issue by mattsm** [qca/boardfarm#1]
DHCP addresses that the WAN gives the DUT from it's DHCP server are hard coded to a private range of IP addresses

**Comment by mbanders** [comment]
What range of IP addresses should it offer from?  Is that something you think should be set in the config file?

**Comment by mattsm** [comment]
Yes, also keep in mind per site DNS servers in the config. So probably a dict with a default. Could also try to query it from the WAN machine, but that could get too complex

**Comment by mbanders** [comment]
You're suggesting changing these lines, right?  https://github.com/qca/boardfarm/blob/843e1ac43de87b0156666fec81e102f902c6de9a/devices/debian.py#L162-L165

And you're suggesting changing line 169 too, right?

``` python
        self.sendline('          range 192.168.0.10 192.168.0.100;')
```

I'm guessing that's a problem if the router by default also hands out addresses in that range to LAN devices.

**Comment by mattsm** [comment]
Not 169 really at this point... it could be a problem though at some point.
